src/components/pieces.js

```diff
@@ -0,0 +1,104 @@
+class Pawn{
+  constructor( color ){
+    this.color = color
+    this.name = 'Pawn'
+    this.moves = 0
```

Comment:
Out of curiosity (seeking a response to this question for my own learning), why are we keeping track of the number of moves a piece has?